messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -988,40 +1006,101 @@ private boolean isDone() {
                                .allMatch(WorkPackage::isDone);
         }
 
+        private MessageStream.Entry<? extends EventMessage<?>> peekNextEvent() {
```

Comment:
Can't you replace this stuff with the new `MessageStream#peek` method?

Reply:
Oh wow, I don't know why I have forgot about that. I replaced it, but in another pleace. Of course it can!